Makefile

```diff
@@ -0,0 +1,68 @@
+# Ontology-metadata ontology Makefile
+# Jie Zheng
+#
+# This Makefile is used to build artifacts
+# for the IAO ontology metadata.
+#
+
+### Configuration
+#
+# prologue:
+# <http://clarkgrubb.com/makefile-style-guide#toc2>
```

Comment:
I wasn't aware of this. I'll see if we should incorporate into ODK

Reply:
I use this block in pretty much every Makefile I write to ensure the defaults are more sane.